app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -38,11 +39,14 @@ $govuk-cookie-banner-background-new: govuk-colour("white");
   margin-bottom: govuk-spacing(1);
   padding-right: govuk-spacing(4);
   padding-bottom: govuk-spacing(2);
+  color: $govuk-cookie-banner-text-green;
+
+  @include govuk-media-query($from: desktop) {
+    padding-bottom: 0;
```

Comment:
There's a point between two media queries where the space underneath this element is collapsed. Everything below this point has the space, so it feels like it should have it here as well if possible?

Good:

<img width="788" alt="Screen Shot 2019-06-12 at 13 13 45" src="https://user-images.githubusercontent.com/861310/59350521-30da8380-8d14-11e9-90b2-6ee50395c3b1.png">

Less good?

<img width="786" alt="Screen Shot 2019-06-12 at 13 14 03" src="https://user-images.githubusercontent.com/861310/59350530-3afc8200-8d14-11e9-8d50-b04b4e536c9d.png">